### DISCUSSION TITLE
How do I run it

### DISCUSSION THREAD

**Discussion by Chainreactionist** [mchehab/zbar#154] [General]

**Comment by wqh17101** [comment]
what`s your os

**Reply by AlBannaTechno** [reply]
@wqh17101 
Hi, My OS is windows 10 x64, I tried to use the old version from SourceForge and it works.
But when I use the version from this fork `from release page`
I can not run anything
I just found three files
```log
libzbar-0.dll
zbarcam.exe
zbarimg.exe
```
then the app always ask fro additional DLLs, and after I get all required DLLS, it just says 
```
WARNING: no compatible input to output format
...trying again with output disabled
```


Also the old issue, with camera names in window still exist,
if you executed `zbarcam.exe "USB2.0_Camera" --verbose=32`
you will get

```log
_zbar_video_open: searching for camera: USB2.0_Camera (-1)
_zbar_video_open:     [0] Microsoft WDM Image Capture (Win32) - Version:  10.0.19041.1
_zbar_video_open:     [1] not found...
_zbar_video_open:     [2] not found...
_zbar_video_open:     [3] not found...
_zbar_video_open:     [4] not found...
_zbar_video_open:     [5] not found...
_zbar_video_open:     [6] not found...
_zbar_video_open:     [7] not found...
_zbar_video_open:     [8] not found...
_zbar_video_open:     [9] not found...
ERROR: zbar video in _zbar_video_open():
    invalid request: video device not found 'USB2.0_Camera'
ERROR: zbar processor in _zbar_video_open():
    invalid request: video device not found 'USB2.0_Camera'
```

I described this issue in [This Stackoverflow Question](https://stackoverflow.com/questions/66507039/what-is-the-correct-device-camera-name-format-should-passed-to-zbar-zbarsh)

Thank you.
